src/context/TransactionsProvider/TransactionsProvider.tsx

```diff
@@ -84,12 +83,21 @@ export const TransactionsProvider = ({ children }: TransactionsProviderProps): J
                 supportedAccountTypes.map(async accountType => {
                   moduleLogger.debug({ chainId, accountType }, 'subscribing txs')
 
-                  const accountParams = accountType
-                    ? utxoAccountParams(chainId, accountType, 0)
-                    : {}
+                  const bip44Params = adapter?.getBIP44Params({ accountNumber: 0, accountType })
```

Comment:
i created this ticket to handle this case here - we need to be able to support websockets for multiple accounts, i'll take this one, we can merge this for now as it's ticketed and not a regression

https://github.com/shapeshift/web/issues/3043